package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.8.29",
   "description": "Solidity compiler",
   "main": "index.js",
+  "types": "index.d.ts",
```

Comment:
You would normally also want exports array. Also sideEffect: false to enable tree shaking. Happy to follow up with a new pr cleaning this stuff up. They aren't critical but help provide universal compatibility to all tools

Reply:
Thanks, yes, that would be great :)